test/unit_tests/aievec_tests/bf16_exp_lut/bf16_exp_lut_mlir_to_llvm.mlir

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Copyright (C) 2023, Advanced Micro Devices, Inc.
+
+// REQUIRES: valid_xchess_license
+// REQUIRES: peano, peano_and_chess
```

Comment:
This guard (peano_and_chess) doesn't work, and as a result this test is not being run. If I try to run the commands by hand, peano cannot compile `dut_simple.cc` or `lut_based_ops.cpp` because it can't find `<utility>`.

I've also noticed that `aie-opt` is not generating a call to `getExpBf16`, it's leaving `math.exp` untouched.